share/smack/svcomp/utils.py

```diff
@@ -48,15 +48,18 @@ def svcomp_check_property(args):
   if args.svcomp_property:
     with open(args.svcomp_property, "r") as f:
       prop = f.read()
+    from smack.top import VProperty
+    from smack.top import VResult
     if "valid-deref" in prop:
-      args.check = ['memory-safety']
+      args.check = VProperty.MEMORY_SAFETY
```

Comment:
Should this be a list/set?

Reply:
No, `args.check` is an element of class `VProperty`.

Reply:
If I understand it correctly, an enum like this in Python is basically a bitmask. So a list of properties like what we had amounts to a mask having certain bits set.